README.md

```diff
@@ -197,11 +197,249 @@ To check that formatting is valid without actually making changes, run `npm run
+
+## How to add a new page
+
+Choose which existing folder from `docs/` your new page belongs to.
+
+Next, choose the file name. The file name will determine the URL. For example, `start/my-new-page.mdx` results in the URL `start/my-new-page`. Choose a file name that will be stable over the page's lifespan and that is unlikely to clash with other topics. Use `-` rather than `_` as the delimiter. You can also ask for help choosing a name in the GitHub issue or pull request.
+
+If your file will have non-trivial code in it, please create a Jupyter notebook ending in `.ipynb`, rather than an MDX file. We prefer Jupyter notebooks when there is code because we have tests to make sure that the code still executes properly, whereas MDX is not tested.
+
+Once the file is created, you need to add metadata. Run `npm run check:metadata` for instructions on how to do this. (Refer to [Check file metadata](#check-file-metadata))
+
+Finally, add the file to the folder's `_toc.json`, such as `start/_toc.json`. The `title` is what will show up in the left side bar. Note that the `url` leaves off the file extension.
+
+## Images
+
+Images are stored in the `public/images` folder. You should use subfolders to organize the files. For example, images for `start/my-file.mdx` should be stored like `public/images/start/my-file/img1.png`.
+
+To use the image inside an MDX page:
+
+```markdown
+![Your image](/images/build/your-file/your_image.png)
+```
+
+To add an inline images,
+
+```markdown
+Inline ![Inline image](/images/build/your-file/your_image.png) image
+```
+
+To include a caption
+
+```markdown
+![Your image](/images/build/your-file/your_image.png "Image caption")
+```
+
+You can include a version of the image to be with the dark theme. You only need to create an image with the same name ending in `@dark`. So for example, if you have a `sampler.png` image, the dark version would be `sampler@dark.png`. This is important for images that have a white background.
+
+## Math
+
+We use [LaTeX](https://www.latex-project.org) to write math, which gets rendered by the library [KaTeX](https://katex.org).
+
+Inline math expressions should start with `$` and end with `$`, e.g. `$\frac{123}{2}$`.
+
+Multi-line expressions should start with `$$` and end with `$$`:
+
+```markdown
+$$
+L = \frac{123}{2} \rho v^2 S C_1s
+$$
+```
+
+## Tables
+
+Tables are supported: https://www.markdownguide.org/extended-syntax/.
+
+## Comments
+
+Example comment: {/_ Comes from https://qiskit.org/documentation/partners/qiskit_ibm_runtime/getting_started.html _/}
+
+## Collapsible sections
+
+For content that you don't want to show by default, use a collapsible section. The user will need to expand the section to read its contents. Refer to GitHub's guide on [`<details>` and `<summary>`](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections).
+
+## Footnotes
```

Comment:
Do footnotes work like this in .ipynb as well?

Reply:
Looks like yep! I tested this out in the hello world notebook

<img width="173" alt="Screenshot 2023-12-13 at 4 40 49 PM" src="https://github.com/Qiskit/documentation/assets/14852634/25eab192-293e-4874-ba5c-2eef0566590d">

<img width="283" alt="Screenshot 2023-12-13 at 4 41 10 PM" src="https://github.com/Qiskit/documentation/assets/14852634/307136f9-efbb-493d-92c6-a9e1807b4a78">

That's really neat! I wrote the definition of the footnote in a different part of the page in the middle, and the application put it automatically at the very bottom of the page.